include/CMakeLists.txt

```diff
@@ -13,6 +14,10 @@ if(INSTALL_MBEDTLS_HEADERS)
         DESTINATION include/psa
         PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ)
 
+    install(FILES ${psa_driver_headers}
+        DESTINATION include/psa/drivers
```

Comment:
The change in installation instructions needs to be documented in the changelog. This affects all applications that use any part of PSA crypto, even when no (non-builtin) drivers are present.

Reply:
Backed out the change introducing the new folder. It's just going to be delaying the driver work and can be done another time.